doc/api/globals.md

```diff
@@ -333,6 +333,17 @@ A browser-compatible implementation of the `EventTarget` class. See
 
 This variable may appear to be global but is not. See [`exports`][].
 
+## `fetch`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental. Enable this API with the [`--experimental-fetch`][]
```

Comment:
Any reason this can't be experimental without requiring a CLI flag? Even a warning would probably be better DX?

Reply:
Given that the undici repo says this:

> This is experimental and is not yet fully compliant with the Fetch Standard. We plan to ship breaking changes to this feature until it is out of experimental.

I think flagging is a good call. Experimental with just a warning seems better for pushing adoption in non-critical projects to iron out bugs, but this seems to be a bit earlier than that.

Reply:
Landing new globals would be a semver-major change, using an experimental flag allows us to have it landed in release lines quicker.